tests/poms/inventory.page.ts

```diff
@@ -19,15 +26,56 @@ export namespace InventoryService {
 		async close() {
 			await this.page.close();
 		}
+
+		async addProductToCart(addProductButtonTestId: string) {
+			const addProductButton = this.page.locator(
+				`[data-test="${addProductButtonTestId}"]`
+			);
+			await addProductButton.click();
+		}
+
+		async removeProductFromCart(removeProductButtonTestId: string) {
+			const removeProductButton = this.page.locator(
+				`[data-test="${removeProductButtonTestId}"]`
+			);
+			await removeProductButton.click();
+		}
+
+		async sortProductsBy(option: string) {
```

Comment:
Change type of this option argument from string to its own. az, za, lohi, hilo so that it could suggest for usable options.